install_config/web_console_customization.adoc

```diff
@@ -227,6 +227,7 @@ angular
       .when('/about', {
         templateUrl: 'extensions/about/about.html',
         controller: 'AboutController'
+        hawtioPluginLoader.addModule('aboutPageExtension');
```

Comment:
Oh I see. It's shown again in the file too. I should've been more observant. Sorry! Thanks!